components/Starknet/modules/quick_start/pages/set_up_an_account.adoc

```diff
@@ -104,9 +104,14 @@ The `fetch` command supports both Argent X and Braavos smart wallets. Make sure
 
 [source,shell]
 ----
-starkli account fetch <SMART_WALLET_ADDRESS> --output ~/.starkli-wallets/deployer/account.json
+starkli account fetch <SMART_WALLET_ADDRESS> --output ~/.starkli-wallets/deployer/account.json --rpc <YOUR_RPC_ENDPOINT_HERE>
 ----
 
+Starkli is centric around JSON-RPC, and the JSON-RPC provider is considered canonical. Users are strongly recommended to use JSON-RPC. There are a few options to obtain access to a JSON-RPC endpoint:
```

Comment:
The Feeder Gateway is deprecated, and much support will be removed soon, so I wouldn't even mention JSON-RPC here as an option, because there really isn't an alternative.
```suggestion
You can obtain access to a JSON-RPC endpoint in one of the following ways:
```